pkg/activator/handler/filtering_handler_test.go

```diff
@@ -75,3 +75,11 @@ func TestFilteringHandler(t *testing.T) {
 		})
 	}
 }
+
+func mapToHeader(m map[string]string) http.Header {
+	h := http.Header{}
+	for k, v := range m {
+		h.Add(k, v)
+	}
+	return h
+}
```

Comment:
Not quite parsing why this function is needed.  Seems more verbose than what we had before?

Reply:
It fixes the question around header canonicalization. By using the built in methods as they are intended to be used, the header names are properly case insensitive.

If there are strong options about what from the header names should take I'll change them, but last time I asked I didn't get a response.

Reply:
Ah, that makes sense, thanks for clarifying!